plugins/imagebase/plugin.js

```diff
@@ -727,7 +727,7 @@
 		/**
 		 * @property {CKEDITOR.dom.element} wrapper An element created for wrapping the progress bar.
 		 */
-		this.wrapper = CKEDITOR.dom.element.createFromHtml( wrapperHtml || '<div class="cke_loader"></div>' );
+		this.wrapper = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview(wrapperHtml || '<div class="cke_loader"></div>', 'wrapperHtml is set by the developer. It is used as a way to allow custom loader htmls for image uploading. The alternative is a safe constant.') );
```

Comment:
Sounds good! (nit: Maybe clarify that "set by the developer" is "not user controlled"?)

Reply:
SGTM